src/components/Api.js

```diff
@@ -0,0 +1,104 @@
+export default class Api {
+  constructor(options) {
+    this._baseUrl = options.baseUrl;
+    this._headers = options.headers;
+  }
+
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
+      headers: this._headers,
+    }).then((res) => {
```

Comment:
Needs correction: The code for processing server response gets duplicated. You should declare a method and reuse it

```js
_processResponse = (res) => { 
      if (res.ok) { 
        return res.json(); 
      } 
      return Promise.reject(`Error: ${res.status}`); 
}

getInitialCards() { 
  ...
  .then(this._processResponse)
}
```